test/mix_java_scala_lib.bzl

```diff
@@ -0,0 +1,11 @@
+def _mix_java_scala_lib_impl(ctx):
+    paths = [f.path for f in ctx.files.deps]
```

Comment:
Is this what you’re using internally which fails?

Reply:
Yeah, this is what we're using in our rules that failed before this change. This maps to the `ctx.outputs` of the `scala_library` rule on which this rule depends:

> In other words, use files to access the default outputs of a dependency.

-- https://docs.bazel.build/versions/master/skylark/lib/ctx.html#files